src/client/e2e/spot-tile.spec.ts

```diff
@@ -47,10 +48,10 @@ test.describe("Spot Tile", () => {
     test("When I buy USD/JPY then an Execution screen appears until transaction is performed and tile displays green confirmation screen", async () => {
       await tilePage.locator("[data-testid='Buy-USDJPY']").click()
 
-      const executingSpinner = await tilePage.getByText(/Executing/)
+      const executingSpinner = tilePage.getByText(/Executing/)
```

Comment:
good catch here .. there are so many awaits in the Playwright docs that it must be tempting to slap it on everything.
but as you have called out here - these are returning Locator objects, not Promises .. it is the method on the Locator that is typically async .. e.g. click()